rest_witchcraft/viewsets.py

```diff
@@ -26,6 +26,7 @@ class ModelViewSet(
     mixins.UpdateModelMixin,
     DestroyModelMixin,
     mixins.ListModelMixin,
+    QuerySerializerMixin,
```

Comment:
Hmm, would it be better to have an `ExpandableModelViewSet`? would rather keep this class as similar as possible with drf model viewset.

Reply:
sure. that should work. even if serializers are diff should fail gracefully and simply not expand anything